tests/stage

```diff
@@ -1,131 +1,29 @@
-    TMPFILE=`mktemp`
-    wget https://raw.githubusercontent.com/Bitergia/mordred/master/docker/unified_releases/$RELEASE -O $TMPFILE
-    source $TMPFILE
-    rm $TMPFILE
-    echo $RELEASE > /home/bitergia/release
-else
-    echo "customized release, see requirements.cfg file" > /home/bitergia/release
-fi
-
-# get Arthur repository
-cd /home/bitergia && \
-# git clone https://github.com/grimoirelab/arthur.git && \
-git clone https://github.com/acs/arthur.git && \
-    cd arthur && \
-    git checkout $ARTHUR && \
-    sudo python3 setup.py install
-
-# get SortingHat repository
-cd /home/bitergia && \
-git clone https://github.com/grimoirelab/sortinghat.git && \
-    cd sortinghat && \
-    git checkout $SORTINGHAT && \
-    sudo python3 setup.py install
-
-# get GrimoireELK repository
-cd /home/bitergia && \
-git clone https://github.com/grimoirelab/GrimoireELK && \
-    cd GrimoireELK && \
-    git checkout $GRIMOIREELK
-
-# get Mordred repository
-cd /home/bitergia && \
-git clone https://github.com/Bitergia/mordred.git mordred && \
-    cd mordred && \
-    git checkout $MORDRED
-
-# panels for the dashboard
-cd /home/bitergia && \
-git clone https://github.com/grimoirelab/panels.git panels && \
-    cd panels && \
-    git checkout $PANELS
-
-# reports repository
-cd /home/bitergia && \
-git clone https://github.com/grimoirelab/reports.git reports && \
-    cd reports && \
-    git checkout $REPORTS
-
-# get GrimoireLab Toolkit repository
-cd /home/bitergia && \
-git clone https://github.com/grimoirelab/grimoirelab-toolkit.git && \
-    cd grimoirelab-toolkit && \
-    git checkout $GRIMOIRELAB_TOOLKIT && \
-    sudo python3 setup.py install
-
-# get Perceval repository
-cd /home/bitergia && \
-    git clone https://github.com/grimoirelab/perceval.git && \
-    cd perceval && \
-    git checkout $PERCEVAL && \
-    sudo python3 setup.py install
-
-# perceval extras
-cd /home/bitergia && \
-    git clone https://github.com/grimoirelab/perceval-mozilla && \
-    cd perceval-mozilla && \
-    git checkout $PERCEVAL_MOZILLA && \
-    sudo python3 setup.py install
-
-cd /home/bitergia && \
-    git clone https://github.com/grimoirelab/perceval-puppet && \
-    cd perceval-puppet && \
-    git checkout $PERCEVAL_PUPPET && \
-    sudo python3 setup.py install
-
-cd /home/bitergia && \
-    git clone https://github.com/grimoirelab/perceval-opnfv && \
-    cd perceval-opnfv && \
-    git checkout $PERCEVAL_OPNFV && \
-    sudo python3 setup.py install
-
-
-# some links we need until GrimoireELK and panels can be installed
-ln -s /home/bitergia/GrimoireELK/grimoire /home/bitergia/mordred/grimoire
-ln -s /home/bitergia/GrimoireELK/grimoire_elk /home/bitergia/mordred/grimoire_elk
-ln -s /home/bitergia/panels /home/bitergia/mordred/panels
-ln -s /home/bitergia/panels/dashboards5 /home/bitergia/panels/dashboards
+echo "Installing and checking GrimoireLab"
+sudo grimoirelab/utils/build_grimoirelab --build --install --check --install_system
```

Comment:
(After our discussion ;-) ). Yes, you can. For now, it can only use master/HEAD in all repos. But II will submit a PR, using an environment variable or something, so you can specify a release file.

Reply:
Great!